backend/src/main/scala/cromwell/backend/io/JobPaths.scala

```diff
@@ -10,26 +10,30 @@ object JobPaths {
   val CallPrefix = "call"
   val ShardPrefix = "shard"
   val AttemptPrefix = "attempt"
+  val CacheCopyPrefix = "cacheCopy"
   val ScriptPathKey = "script"
   val StdoutPathKey = "stdout"
   val StdErrPathKey = "stderr"
   val ReturnCodePathKey = "returnCode"
   val CallRootPathKey = "callRootPath"
   val DockerCidPathKey = "dockerCidPath"
 
-  def callPathBuilder(root: Path, jobKey: JobKey) = {
+  def callPathBuilder(root: Path, jobKey: JobKey, isCallCacheCopyAttempt: Boolean) = {
     val callName = jobKey.node.localName
     val call = s"$CallPrefix-$callName"
     val shard = jobKey.index map { s => s"$ShardPrefix-$s" } getOrElse ""
-    val retry = if (jobKey.attempt > 1) s"$AttemptPrefix-${jobKey.attempt}" else ""
 
-    List(call, shard, retry).foldLeft(root)((path, dir) => path.resolve(dir))
+    val retryOrCallCache = if (isCallCacheCopyAttempt) CacheCopyPrefix else s"$AttemptPrefix-${jobKey.attempt}"
+    List(call, shard, retryOrCallCache).foldLeft(root)((path, dir) => path.resolve(dir))
```

Comment:
NB: these two lines are the big changes. Most of the other stuff is about passing in the right values.